core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java

```diff
@@ -390,6 +390,8 @@ public void provideOutcome(AppCreator ctx) throws AppCreatorException {
             if (additionalBuildArgs != null) {
                 command.addAll(additionalBuildArgs);
             }
+            command.add("--initialize-at-build-time=");
```

Comment:
When you don't have a value, it initializes everything at build time?

Reply:
Right you have to give an empty value to indicate that build time init is the default.